policyengine_us/variables/gov/irs/credits/retirement_savings/retirement_saving.py

```diff
@@ -0,0 +1,45 @@
+from policyengine_us.model_api import *
+
+
+class retirement_saving_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    label = "Retirement Savings Credit"
+    unit = USD
+    reference = "https://www.irs.gov/pub/irs-pdf/f8880.pdf"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.credits.retirement_saving
+        person = tax_unit.members
+        person_eligible = person("retirement_saving_eligible_person", period)
+        agi_person = (
+            person("adjusted_gross_income_person", period) * person_eligible
```

Comment:
do eligibility separately to avoid an "AGI but zero if ineligible" paradigm (this number can be confused with having zero AGI)